webpack/scenes/ContentViews/Copy/CopyContentViewModal.js

```diff
@@ -5,17 +5,27 @@ import CopyContentViewForm from './CopyContentViewForm';
 
 const CopyContentViewModal = ({
   cvId, cvName, show, setIsOpen,
-}) => (
-  <Modal
-    title={`Copy content view ${cvName}`}
-    variant={ModalVariant.large}
-    isOpen={show}
-    width="50%"
-    onClose={() => { setIsOpen(false); }}
-    appendTo={document.body}
-  ><CopyContentViewForm cvId={cvId} setModalOpen={setIsOpen} />
-  </Modal>
-);
+}) => {
+  const description = (
+    <p>
+      This will create a copy of <b>{cvName}</b>, including details,
+      repositories, and filters.<br />Generated data such
+      as history, tasks and versions will not be copied.
+    </p>
```

Comment:
The wrapping looks a little odd here as it will split into two sentences if the name is long enough:

![here](http://www.clipular.com/c/5293184537853952.png?k=15fC_tRdwu3X8_dnyF-qpdfbSNI)

You can use backticks and backslash to make a long string and split over lines, removing the `br` to let it wrap as needed. I think `<b>` can be included in the string and still render, but we can come up with something if that is not the case!

```js
`testing that \
this should be \
one long string`
-> "testing that this should be one long string"
```

Reply:
Should be a `<strong>` instead, see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b :)